tests/unitTests/finiteVolume/processes/twoPhaseEulerAdvectionTests.cpp

```diff
@@ -181,19 +181,19 @@ INSTANTIATE_TEST_SUITE_P(TwoPhaseEulerAdvectionTests, TwoPhaseEulerAdvectionTest
                                  .conservedValuesIn = {340.4, 851000000.0, 3404.0, -6808.0, 10212.0, 0.8},  // RHO, RHOE, RHOU, RHOV, RHOW, RHOALPHA
                                  .normalIn = {0.5, 0.5, 0.7071},                                            // x, y, z
                                  .expectedDensity = 340.4,
-                                 .expectedDensityG = 1.2352213315429057,
-                                 .expectedDensityL = 963.834108739261,
+                                 .expectedDensityG = 1.2352213315429055,
```

Comment:
Do any of these tests test StiffenedGasStiffenedGasDecoder?

Reply:
No, I will add some tests for that